UnitTestsHostApp/SceneDelegate.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SceneDelegate.swift
+//
+//  Created by Andrés Boedo on 9/24/21.
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

Comment:
I must have missed committing this file when I created the app, and for iOS < 15, the app would crash on start because of it. 

That said, it still crashes on startup on iOS < 15, but this time it's because of the StoreKit 2 thing 😂  (which we filed a radar for)